mapboxgl/templates/vector.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+
+{% block legend %}{% endblock legend %}
+
+{% block map %}
+
+    map.on('style.load', function() {
+        let params = {
+            "type": "vector",
+            "tiles": [
+                 "{{ tiles_url }}"
+            ],
+            "minzoom": {{ tiles_minzoom }},
+            "maxzoom": {{ tiles_maxzoom }}
+        }
+        if ({{ tiles_bounds }} !== undefined) params.bounds = {{ tiles_bounds }};
```

Comment:
Do we want this to happen even if `tiles_bounds` is null (i.e., do we treat `null` separately from `undefined` here)? Because this might make sense as `if ({{ tiles_bounds }} != null)` (which will match both `null` *and* `undefined`).